tests/test-vip-plugins.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace Automattic\VIP\Tests;
+
+class VIP_Go_Plugins_Test extends \WP_UnitTestCase {
+
+	protected $option_active_plugins = [];
+	protected $option_active_sitewide_plugins = [];
+	protected $code_activated_plugins = [];
+
+	public function setUp() {
+		parent::setUp();
+
+		// emulate the active plugins option
+		$this->option_active_plugins = [
+			'hello.php',
+			'airplane-mode/airplane-mode.php',
+			'msm-sitemap/msm-sitemap.php',
+			'gutenberg/gutenberg.php',
+			'liveblog/liveblog.php',
+			'wp-instagram-widget/wp-instagram-widget.php',
+			'shortcake/shortcode-ui.php',
+		];
+
+		// emulate the network active plugins option
+		$this->option_active_sitewide_plugins = [
+			'hello.php' => 1507904134,
+			'akismet/akismet.php' => 1507904154,
+			'msm-sitemap/msm-sitemap.php' => 1507904154,
+			'shortcake/shortcode-ui.php' => 1507904134,
+		];
+
+		// emulate the code activated plugins array
+		$this->code_activated_plugins = [
+			'shared-plugins/two-factor/two-factor.php',
+			'plugins/advent-calender/advent-calendar.php',
+			'plugins/msm-sitemap/msm-sitemap.php',
+			'shared-plugins/vip-go-elasticsearch/vip-go-elasticsearch.php',
+		];
+	}
+
+	public function test__modify_active_plugins() {
+		/**
+		 * Set active_plugins to empty to start the test
+		 */
+		update_option( 'active_plugins', [] );
+
+		/**
+		 * Ensure the values are indeed empty
+		 */
+		$this->assertEmpty( get_option( 'active_plugins' ) );
+
+		/**
+		 * Update the option with our list of active plugins
+		 */
+		update_option( 'active_plugins' , $this->option_active_plugins );
+
+		/**
+		 * Check that option is not empty
+		 */
+		$this->assertNotEmpty( get_option( 'active_plugins' ) );
```

Comment:
We should actually confirm that the value of `active_plugins` in the db matches what we expect.